packages/babel-core/src/transformation/file/file.ts

```diff
@@ -151,6 +153,13 @@ export default class File {
   }
 
   addHelper(name: string): t.Identifier {
+    if (helpers.isInternal(name)) {
+      throw new Error("Cannot use internal helper " + name);
+    }
+    return this.#addHelper(name);
```

Comment:
Since this file will be compiled to Node.js 6, can we turn on the `privateFieldsAsProperties` assumption in the build config?

Reply:
I'm using `privateFieldsAsSymbols` instead, just to avoid potential conflicts.